pkg/messaging/client.go

```diff
@@ -41,17 +36,7 @@ func StartMessagingClient(ctx context.Context, nanomsgURL string, bot *telebot.B
 					log.Printf("failed to receive message: %v", err)
 					return
 				}
-				log.Printf("message received: %s", string(msg))
-
-				if !botEnv.ReceivedChat {
-					log.Println("haven't received a chat id yet")
-					continue
-				}
-
-				_, err = bot.Send(botEnv.Chat, string(msg))
-				if err != nil {
-					log.Printf("failed to send a message to telegram, %v", err)
-				}
+				bots.TgBotEnvironment.SendMessageTg(msg)
```

Comment:
`Bots` is so telegram specific. Better make `Bots` a slice of `Bot` interfaces with `SendMessage` method and call it here for every bot in collection.